playwright-test/playwright-local.config.js

```diff
@@ -0,0 +1,87 @@
+// playwright.config.js
+// @ts-check
+const { devices } = require('@playwright/test');
+
+/** @type {import('@playwright/test').PlaywrightTestConfig} */
+const config = {
+  testDir: 'tests',
+  testMatch: '**/*.spec.js',
+  // Use globalSetup & globalTearedown only if browserstack.local = true
+  globalSetup: require.resolve('./global-setup'),
+  globalTeardown: require.resolve('./global-teardown'),
+  timeout: 60000,
+  projects: [
+    // -- BrowserStack Projects --
+    // name should be of the format browser@browser_version:os os_version@browserstack
+    {
+      name: 'chrome@latest:Windows 10@browserstack',
+      use: {
+        browserName: 'chromium',
+        channel: 'chrome'
+      },
+    },
+    {
+      name: 'chrome@latest-beta:OSX Big Sur@browserstack',
+      use: {
+        browserName: 'chromium',
+        channel: 'chrome',
+      },
+    },
+    {
+      name: 'edge@90:Windows 10@browserstack',
+      use: {
+        browserName: 'chromium',
+        viewport: { width: 1280, height: 1024 }
+      },
+    },
+    {
+      name: 'playwright-firefox@latest:OSX Catalina@browserstack',
+      use: {
+        browserName: 'firefox',
+        viewport: null,
+        ignoreHTTPSErrors: true
+      },
+    },
+    {
+      name: 'playwright-webkit@latest:OSX Big Sur@browserstack',
+      use: {
+        browserName: 'webkit',
+        ...devices['iPhone 12 Pro Max'],
```

Comment:
comment out this line and mention a second comment that this is how users can use Playwright's emulated devices